samples/core/Miscellaneous/Testing/TestProject/TestProject.csproj

```diff
@@ -1,185 +1,21 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{F4468750-9480-48E0-AA7A-DE84780AE1C2}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>EFProviders.InMemory</RootNamespace>
-    <AssemblyName>EFProviders.InMemory</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <ProjectTypeGuids>{3AC096D0-A1C2-E12C-1390-A8335801FDAB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">10.0</VisualStudioVersion>
-    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
-    <ReferencePath>$(ProgramFiles)\Common Files\microsoft shared\VSTT\$(VisualStudioVersion)\UITestExtensionPackages</ReferencePath>
-    <IsCodedUITest>False</IsCodedUITest>
-    <TestProjectType>UnitTest</TestProjectType>
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
-    <TargetFrameworkProfile />
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
@bricelam Should we make the default TFM a variable as well?

Reply:
If it were up to me, I'd just keep it at the lowest compatible framework and only update it as needed.

Reply:
> @bricelam Should we make the default TFM a variable as well?

If we make the EF Core version a variable, I don't see why not. 

That said I think using variable here is a bit weird. On one hand I see that it makes the samples easier to update in the future, but:
- Each sample is no longer a self-contained simple app
- Code in the sample will often be EF Core version specific anyway

> If it were up to me, I'd just keep it at the lowest compatible framework and only update it as needed.

So in this case all samples should be EF Core 3.0, the minimum compatible will be netcoreapp3.0 (or netstandard2.1), correct?

Reply:
I had the same hesitation when extracting nuget dependency versions to a solution Directory.Build.props - should each csproj stand on its own for ease-of-use etc. I'll revert that and have every csproj contain its own versions as before (3.0), it's not too much trouble to update.

Reply:
Note: will update from 3.0.0-rc1 to 3.0.0 RTM after release (#1760)